src/forms-opportunities/oppty-handlers/accessibility-handler.js

```diff
@@ -274,21 +271,12 @@ export default async function handler(message, context) {
       log.info(`[Form Opportunity] [Site Id: ${siteId}] A11y opportunity not detected, skipping guidance`);
       return ok();
     }
-    const site = await Site.findById(siteId);
-    // send message to mystique for guidance
-    const mystiqueMessage = {
-      type: 'guidance:forms-a11y',
-      siteId,
-      auditId,
-      deliveryType: site.getDeliveryType(),
-      time: new Date().toISOString(),
-      data: {
-        url: site.getBaseURL(),
-        opportunityId: opportunity?.getId(),
-      },
-    };
-    await sqs.sendMessage(env.QUEUE_SPACECAT_TO_MYSTIQUE, mystiqueMessage);
-    log.info(`[Form Opportunity] [Site Id: ${siteId}] Sent a11y message to mystique for guidance`);
+
+    log.info(`[Form Opportunity] [Site Id: ${siteId}] a11y opportunity: ${JSON.stringify(opportunity, null, 2)}`);
+    const opportunityData = opportunity.getData();
+    const a11yData = opportunityData.accessibility;
+    const formsList = a11yData.map((item) => ({ form: item.form, formSource: item.formSource }));
+    await sendMessageToFormsQualityAgent(context, opportunity, formsList);
```

Comment:
A separate task. Can we check if the form is already sent to mystique for gathering form details.
I haven't seen form (or at least the intent) changing a lot over a period.